### ISSUE TITLE
Skaffold File Indexing: Throwable: It's prohibited to access index during event dispatching

### ISSUE THREAD

**Issue by etanshaul** [GoogleCloudPlatform/cloud-code-intellij#2672]
`java.lang.Throwable at Logger.error : Read access is allowed from event dispatch thread or inside read-action only (see com.intellij.openapi.application.Application.runReadAction())`
```
java.lang.Throwable: It's prohibited to access index during event dispatching
	at com.intellij.openapi.diagnostic.Logger.error(Logger.java:145)
	at com.intellij.openapi.project.NoAccessDuringPsiEvents.checkCallContext(NoAccessDuringPsiEvents.java:39)
	at com.intellij.util.indexing.FileBasedIndexImpl.ensureUpToDate(FileBasedIndexImpl.java:736)
	at com.intellij.util.indexing.FileBasedIndexImpl.processExceptions(FileBasedIndexImpl.java:907)
	at com.intellij.util.indexing.FileBasedIndexImpl.processValueIterator(FileBasedIndexImpl.java:983)
	at com.intellij.util.indexing.FileBasedIndexImpl.processValuesInScope(FileBasedIndexImpl.java:954)
	at com.intellij.util.indexing.FileBasedIndexImpl.getContainingFiles(FileBasedIndexImpl.java:828)
	at com.intellij.psi.search.FileNameIndexServiceImpl.getFilesWithFileType(FileNameIndexServiceImpl.java:42)
	at com.intellij.psi.search.FileTypeIndex.getFiles(FileTypeIndex.java:41)
	at com.google.cloud.tools.intellij.kubernetes.skaffold.SkaffoldFileService.findSkaffoldFiles(SkaffoldFileService.kt:99)
	at com.google.cloud.tools.intellij.kubernetes.skaffold.SkaffoldConfigurationDetector$Companion.isSkaffoldConfigurationNeeded(SkaffoldConfigurationDetector.kt:70)
	at com.google.cloud.tools.intellij.kubernetes.skaffold.SkaffoldConfigurationDetector.showPromptForSkaffoldConfigurations(SkaffoldConfigurationDetector.kt:130)
	at com.google.cloud.tools.intellij.kubernetes.skaffold.SkaffoldConfigurationDetector.access$showPromptForSkaffoldConfigurations(SkaffoldConfigurationDetector.kt:52)
```

description of task from user:
> Git update, switched branches, caused reindex.

plugin 19.11.1
IDEA 2019.3 (EAP)

**Comment by etanshaul** [comment]
this seems to be new to 2019.3.

**Comment by etanshaul** [comment]
issue is fixed in the 19.12.1 release
